apps/web/src/pages/EmployeeProfile/components/CareerDevelopmentSection/CareerDevelopmentSection.tsx

```diff
@@ -0,0 +1,503 @@
+    careerDevelopmentOptions?.mentorship,
+  ).map(({ value }) => ({
+    value,
+    label: intl.formatMessage(getMentorshipInterest(value)),
+  }));
+
+  const execCoachingStatusOptions: Radio[] = [
+    { value: ExecCoachingStatus.NOT_PARTICIPATING },
+    ...unpackMaybes(careerDevelopmentOptions?.execCoaching),
+    { value: ExecCoachingStatus.LEARNING_AND_COACHING },
+  ].map(({ value }) => ({
+    value,
+    label: intl.formatMessage(getExecCoachingStatus(value)),
+  }));
+
+  const execCoachingInterestOptions: Radio[] = unpackMaybes(
+    careerDevelopmentOptions?.execCoaching,
+  ).map(({ value }) => ({
+    value,
+    label: intl.formatMessage(getExecCoachingInterest(value)),
+  }));
+
+  return (
+    <ToggleSection.Root
+      id="career-development-form"
+      open={isEditing}
+      onOpenChange={setIsEditing}
+    >
+      <ToggleSection.Header
+        Icon={icon.icon}
+        color={icon.color}
+        level="h3"
+        size="h4"
+        toggle={
+          <ToggleForm.LabelledTrigger
+            sectionTitle={intl.formatMessage({
+              defaultMessage: "Career development preferences",
+              id: "Ey8cB4",
+              description: "Title for career development preferences section",
+            })}
+          />
+        }
+        data-h2-font-weight="base(bold)"
+      >
+        {intl.formatMessage({
+          defaultMessage: "Career development preferences",
+          id: "Ey8cB4",
+          description: "Title for career development preferences section",
+        })}
+      </ToggleSection.Header>
+      <p>{subtitle}</p>
+      <ToggleSection.Content>
+        <ToggleSection.InitialContent>
+          {isNull ? (
+            <ToggleForm.NullDisplay />
+          ) : (
+            <Display
+              employeeProfile={employeeProfile}
+              careerDevelopmentOptions={careerDevelopmentOptions}
+            />
+          )}
+        </ToggleSection.InitialContent>
+        <ToggleSection.OpenContent>
+          <BasicForm
+            labels={labels}
+            onSubmit={handleSave}
+            options={{
+              defaultValues: dataToFormValues(employeeProfile),
+            }}
+          >
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1)"
+            >
+              <Checklist
+                idPrefix="organizationTypeInterest"
+                legend={labels.organizationTypeInterest}
+                name="organizationTypeInterest"
+                id="organizationTypeInterest"
+                items={organizationTypeInterestOptions}
+              />
+              <Checklist
+                idPrefix="moveInterest"
+                legend={labels.moveInterest}
+                name="moveInterest"
+                id="moveInterest"
+                items={moveInterestOptions}
+              />
+              <Separator data-h2-margin="base(0)" />
```

Comment:
Need the decorative prop here

Reply:
Always forget this 😅 [5d09538](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12680/commits/5d09538675cde5129e1d0480f266af18256951a4)